example.js

```diff
@@ -1,3 +1,5 @@
+var local = require('os').hostname() + '.local'
```

Comment:
@mafintosh on OS X the `os.hostname()` already includes the `.local` part - I'm not sure if this differs between operation systems, but I think the `+ '.local'` part should be removed